keycloak/themes/hmda/login/template.ftl

```diff
@@ -98,10 +98,15 @@
     </div>
   </#if>
 
-  <div class="usa-grid usa-grid-small">
+  <#assign className="usa-grid usa-grid-small"/>
+  <#if (displayMessage && message?has_content && message.summary == "verifyEmailMessage")>
```

Comment:
Just from my overall comment ... is there a reason to make this message different from others here?

Reply:
The reason this is separated here is to separate out the help text (which is automatically inserted in the body of all other error messages, whereas, per #164, the help text was out of the error box.

Reply:
👍